umade-client/src/pages/CustomerUsecases/ViewPaidProducts.jsx

```diff
@@ -0,0 +1,106 @@
+import React, {useEffect, useState} from "react";
+import axios from 'axios';
+import './ViewPaidProduct.css'; 
+import Box from '@mui/material/Box';
+import TextField from '@mui/material/TextField';
+import HomeIcon from '@mui/icons-material/Home';
+import LocalOfferSharpIcon from '@mui/icons-material/LocalOfferSharp';
+import StorefrontSharpIcon from '@mui/icons-material/StorefrontSharp';
+
+function ViewPaidProduct(){
+    const [data, setData] = useState([]);
+    const [id, setId] = useState(0);
+    const [name, setName] = useState('');
+    const [description, setDescription] = useState('');
+    const [tags, setTags] = useState('');
+    const [image, setImage] = useState('');
+    const [price, setPrice] = useState(0);
+    const [amount, setAmount] = useState(0);
+    const [sellBy, setSellBy] = useState(0);
+    const [createdDate, setCreatedDate] = useState(0);
+    const [updatedDate, setUpdatedDate] = useState(0);
+    const [isDeleted, setIsDeleted] = useState(0);
+    const [sellerName, setSellerName] = useState('');
+    const [product, setProduct] = useState({
+        id: "",
+        name: "",
+        description: "",
+        tags: "",
+        image: "",
+        price: "",
+        amount: "",
+        sellBy: "",
+        createdDate: "",
+        updatedDate: "",
+        isDeleted: "",
+        sellerName: ""
+    })
+
+    useEffect(() => {
+        axios.get("http://localhost:8080/api/paidProduct").then((res) => setData(res.data)).catch((err) => alert(err))
+    }, []);
+
+    console.log(data);
+
+    return(
+        <div className="page">
+            
+            <div className="header">
+                <div className="logo">
+                    <img src="./img/Picture1.png" alt="Logo" />
+                </div>
+
+                <div className="search">
+                    <Box
+                        component="form"
+                        sx={{
+                            '& > :not(style)': { m: 1, width: '50ch'},
+                        }}
+                        noValidate
+                        autoComplete="off"
+                        >
+                        <TextField id="outlined-basic" label="Tìm kiếm..." variant="outlined"/>
+                    </Box>
+                </div>
+            </div>
+
+            <div className="menu">
+                <ul>
+                    <li><HomeIcon/><a href="#">  Trang chủ</a></li>
+                    <li><a href="#">Sản phẩm</a></li>
+                    <li><a href="#">Cửa Hàng</a></li>
+                    <li><a href="#">Liên hệ</a></li>
+                </ul>
+		    </div>
+
+
+            <div className="title">
+                <h2>DANH SÁCH SẢN PHẨM ĐÃ ĐƯỢC THANH TOÁN</h2>
+            </div>
+            <div className= 'container'>
+                {data.length > 0 ? data.map((product) => {
+                    return (
+                            <div className= 'product'>
+                                <div className="prd_img">
```

Comment:
HTML tag's class naming should be kebab case
```suggestion
                                <div className="prd-img">
```